velox/expression/tests/SimpleFunctionTest.cpp

```diff
@@ -1508,6 +1508,13 @@ struct NoThrowFunction {
     out = in / 6;
     return Status::OK();
   }
+
+  Status callNullable(out_type<int64_t>& out, const arg_type<int64_t>* in) {
```

Comment:
nit: It feels like an anti-pattern to define both call and callNullable. I wonder if it would be cleaner to define a separate test function and a separate TEST_F method.